### ISSUE TITLE
[Screen Reader – Make Code - Home page] Footer landmark is not defined under ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3513]
**User Experience**:
If the landmarks is not defined, then it will hinder the visually impacted users in navigating the application and also make difficulties to understand the application.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that footer landmark is defined on home page or not.

**Actual Result:**
Footer landmark is not defined under ‘Home page’.

**Expected Result:**
Footer landmark should be define under ‘Home page’.

**Note:**
Same issue exists under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.4.1 - Bypass Blocks](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#A11yMAS, #Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.1, #Benchmark, #Win10-Edge(Chromium)

![MAS2 4 1_Footer landmark is not defined](https://user-images.githubusercontent.com/72871089/97545371-e9c85d80-19f0-11eb-8f7c-e44e6a29000c.jpg)
